doc/holochain_101/src/how_to_contribute.md

```diff
@@ -0,0 +1,102 @@
+
+There are two main pathways to contribute. One is by editing files directly in GitHub, and the other is cloning the repository to your computer, running mdbook locally, and then submitting change requests. The following covers both scenarios.
+
+## Writing Guidelines
+
+Please do not edit the SUMMARY.md file, which stores the raw chapter structure, without advanced signoff from the documentation team.
+
+More forthcoming!
+
+## How the Book Works
+Writing is in classic markdown, so a new page is a new markdown file. These files get rendered in the book panel. One markdown file, SUMMARY.md stores the structure of the book as an outline.
+
+The HTML files used in the Book get automatically built from the markdown files. Even though they're auto-generated, static HTML files, one can search within the text.
+
+## Contributing via GitHub
+
+### Getting there
+
+1) Log on to your GitHub account
+
+2) In the Holochain Rust repo, everything is under the `doc/holochain_101/src` folder. All markdown files are there, and some are nested in subfolders. Navigate to the following link: [https://github.com/holochain/holochain-rust/tree/develop/doc/holochain_101/src](https://github.com/holochain/holochain-rust/tree/develop/doc/holochain_101/src)
+
+3) Determine whether you are making, editing, or reviewing an article.
+
+### Access Rights
+
+Please note that while those with direct write access to the repository will be able to submit change requests without creating a fork, those without will have to create a fork. However, this actually adds very little complexity in practice, just meaning that those with write access to the repository will have to review and merge your pull request.
+
+### Making a new article
+
+1) Click "Create New File"
+
+2) Name this file what you intend to name the article, plus the `.md` extension, i.e. `how_to_contribute.md`
+
+3) Use classic markdown to set up the page title, i.e. "# How to contribute"
+
+4) Write the rest of your text, checking the "Preview" tab to see how it would look.
+
+5) Scroll to the bottom of the page and select the option "create a new branch for this commit and start a pull request". You can name a branch, though GitHub will set one automatically. If you know it, mention the issue that the request addresses.
+
+6) Click "Propose New File". Proceed to the Opening a Pull Request section.
+
+### Editing an article you created
+
+1) Navigate to the article you want to edit, making sure you are on the branch where it was first created. Check which branch you are on by looking for the "Branch: ?" dropdown. Use the dropdown to switch to your branch if you're on the wrong one.
```

Comment:
is this right? i expected that after an article is merged we create new branches for new updates

Reply:
This is how Connor explained the process...it may make more sense to make new branches for new updates once the contributions are coming more from outside the team, though.

Reply:
We could divvy up these instructions a little bit more. There's two scenarios:
1. they've made one change, such as create a file, and want to make additional changes
--- implication: they need to be on the same branch as their initial change
2. they've made no previous changes on a branch, and just want to edit an article
--- implication: they just have to be on the `develop` main branch and click `edit`. Nothing special.